### PR TITLE
DrivAerML: Fourier+no-EMA — Lion optimizer vs AdamW

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
On TandemFoil, Lion optimizer with lr=3e-4 significantly outperforms AdamW — it is the optimizer used in every TandemFoil best result. On DrivAerML, the current best uses AdamW lr=5e-4, but this has never been directly compared against Lion in the Fourier+no-EMA configuration. Lion uses sign-based updates which may be better suited to the 3D aerodynamic geometry of DrivAerML. Testing Lion lr=3e-4 and lr=1e-4 against the AdamW baseline.

## Instructions

Run two parallel jobs:

**Run 1 — Lion lr=3e-4:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=0 python train.py \
  --dataset drivaerml \
  --optimizer lion \
  --lr 3e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group asuka-drivaerml-lion &
```

**Run 2 — Lion lr=1e-4:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=1 python train.py \
  --dataset drivaerml \
  --optimizer lion \
  --lr 1e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group asuka-drivaerml-lion
```

Wait for both to complete. Report `val_primary/surface_rel_l2_pct` for each run, epochs completed, and W&B run IDs. Compare against AdamW baseline (51.35%).

**Critical:** You must pass `--epochs 999` — the default is only 2 epochs due to a hardcoded bug in the training script.

## Baseline
- **Primary metric:** `val_primary/surface_rel_l2_pct` (lower is better)
- **Current best (val):** 51.35% with AdamW lr=5e-4 (PR #2475, W&B run: 5ncrjm32)
- **Current best (test):** 52.06%
- **TandemFoil precedent:** Lion lr=3e-4 is the best optimizer across all TandemFoil experiments
- **Reproduce baseline:** `cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2529 because Duplicate assignment — closing extra PR created by parallel agent. Student already has a primary experiment assigned..

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2529 because Closing duplicate — student asuka already has active PR #2521 assigned..
